service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java

```diff
@@ -2226,8 +2226,17 @@ public void updateOrderAdminPageInfo(UpdateOrderPageAdminDto updateOrderPageDto,
                     lang,
                     currentUser);
             }
+            if (nonNull(updateOrderPageDto.getUpdateResponsibleEmployeeDto())) {
+                updateOrderPageDto.getUpdateResponsibleEmployeeDto().stream()
+                    .forEach(dto -> ordersAdminsPageService.responsibleEmployee(List.of(orderId),
+                        dto.getEmployeeId().toString(),
+                        dto.getPositionId(),
+                        currentUser));
+            }
         } catch (UnexistingOrderException | PaymentNotFoundException | UserNotFoundException | UBSuserNotFoundException
-            | NotFoundOrderAddressException | ReceivingStationNotFoundException | OrderNotFoundException e) {
+            | NotFoundOrderAddressException | ReceivingStationNotFoundException | OrderNotFoundException
```

Comment:
too many exceptions. I'd rather catch "Exception" than defining all of that.

Reply:
Fixed